code/controllers/subsystems/initialization/customitems.dm

```diff
@@ -79,45 +73,46 @@ SUBSYSTEM_DEF(customitems)
 		place_custom_item(M,citem)
 
 /datum/custom_item
-	var/ckey
+	var/character_ckey
 	var/character_name
-	var/item_icon_state
 	var/item_desc
 	var/item_name
 	var/item_path
+	var/item_icon
+	var/item_state
 	var/apply_to_target_type
 	var/list/req_access
 	var/list/req_titles
 	var/list/additional_data
 
 /datum/custom_item/New(var/list/data)
-	ckey                 = ckey(data["ckey"])
-	character_name       = lowertext(data["character_name"])
-	item_name            = data["item_name"]
-	item_desc            = data["item_desc"]
-	item_icon_state      = data["item_icon_state"]
-	item_path            = text2path(data["item_path"])
-	req_access           = data["req_access"]                      || list()
-	req_titles           = data["req_titles"]                      || list()
-	additional_data      = data["additional_data"]                 || list()
-	apply_to_target_type = text2path(data["apply_to_target_type"]) || data["apply_to_target_type"]
+
+	character_ckey =       data["character_ckey"]
+	character_name =       data["character_name"]
+	item_name =            data["item_name"]
+	item_desc =            data["item_desc"]
+	item_icon =            data["item_icon"]
+	item_state =           data["item_state"]
+	req_access =           data["req_access"]
+	req_titles =           data["req_titles"]
+	item_path =            data["item_path"]
+	additional_data =      data["additional_data"]
+	apply_to_target_type = data["apply_to_target_type"]
+
+	try
+		character_ckey =       ckey(character_ckey)
+		character_name =       lowertext(character_name)
+		item_path =            item_path && text2path(item_path)
+		apply_to_target_type = apply_to_target_type && text2path(apply_to_target_type)
+		item_icon =            item_icon && file(item_icon)
+	catch(var/exception/E)
+		to_world_log("Exception while loading custom item: [E]")
```

Comment:
This is sort of confusing, as it doesn't actually throw the error (meaning it's logged differently from a runtime). Simpler handling may be to just let it runtime.

Reply:
Done.